yb-voyager/src/migassessment/sizing.go

```diff
@@ -43,38 +44,129 @@ type SourceDBMetadata struct {
 	Size            float64        `db:"size_in_bytes,string"`
 }
 
-var baseDownloadPath = "src/migassessment/resources/remote/"
-var DB *sql.DB
-var SourceMetaDB *sql.DB
+type ExpDataColocatedLimit struct {
+	maxColocatedSizeSupported  sql.NullFloat64 `db:"max_colocated_db_size_gb,string"`
+	numCores                   sql.NullFloat64 `db:"num_cores,string"`
+	memPerCore                 sql.NullFloat64 `db:"mem_per_core,string"`
+	maxSupportedNumTables      sql.NullInt64   `db:"max_num_tables,string"`
+	minSupportedNumTables      sql.NullFloat64 `db:"min_num_tables,string"`
+	maxSupportedSelectsPerCore sql.NullFloat64 `db:"max_selects_per_core,string"`
+	maxSupportedInsertsPerCore sql.NullFloat64 `db:"max_inserts_per_core,string"`
+}
+
+const (
+	COLOCATED_LIMITS_TABLE    = "colocated_limits"
+	SHARDED_SIZING_TABLE      = "sharded_sizing"
+	COLOCATED_LOAD_TIME_TABLE = "colocated_load_time"
+	SHARDED_LOAD_TIME_TABLE   = "sharded_load_time"
+	GITHUB_RAW_LINK           = "https://raw.githubusercontent.com/yugabyte/yb-voyager/main/yb-voyager/src/migassessment/resources"
+)
+
+var ExperimentDB *sql.DB
+var experimentDataFileName = "/yb_2024_0_source.db"
```

Comment:
can remove the `/` here if you are using `filepath.Join()` function everywhere for paths.

Reply:
done